google/cloud/forseti/services/model/service.py

```diff
@@ -133,16 +138,31 @@ def GetModel(self, request, _):
         """
 
         model = self.modeller.get_model(request.identifier)
-
+        created_at_str = self._get_model_created_at_str(model)
         if model:
             return model_pb2.ModelDetails(name=model.name,
                                           handle=model.handle,
                                           status=model.state,
+                                          createdAt=created_at_str,
                                           description=model.description,
                                           message=model.message,
                                           warnings=model.warnings)
         return model_pb2.ModelDetails()
 
+    @staticmethod
+    def _get_model_created_at_str(model):
+        """Get model created_at datetime in string format.
+        Example return value: '2011-11-03 18:21:26'
```

Comment:
I think this might not match what the `DEFAULT_FORSETI_HUMAN_TIMESTAMP`, so might want to drop it.
Instead in the doc string, can just say:
```
Get model created_at datetime in human readable string format.
```